### PR TITLE
Docs and such

### PR DESCRIPTION
**Pull request by marcharper**
Some simple updates to the documentation and more style updates.

Let me know what you think!

### CODE REVIEW

**Comment by drvinceknight** [comment]
Looks awesome to me: I like the addition to the landing page!

Have opened https://github.com/marcharper/Axelrod/pull/5 with a fix for the rst syntax.

File: docs/index.rst [hunk 1]
```diff
@@ -6,7 +6,35 @@
...
+Here is quick overview of the current capabilities of the library:
+* Over 100 strategies from the literature and some exciting original
+contributions
+    * Classic strategies like TiT-For-Tat, WSLS, and variants
+    * Zero-Determinant and other Memory-One strategies
+    * Many generic strategies that can be used to define an array of popular
+    strategies, including finite state machines, strategies that hunt for
+    patterns in other strategies, and strategies that combine the effects of
+    many others
+    * Strategy transformers that augment that abilities of any strategy
```
Comment (drvinceknight) [inline docs/index.rst]: that augment the

File: docs/index.rst [hunk 2]
```diff
@@ -6,7 +6,35 @@
...
+* Head-to-Head matches
+* Round Robin tournaments with a variety of options, including:
+    * noisy environments
+    * spatial games
```
Comment (drvinceknight) [inline docs/index.rst]: spatial tournaments?

File: docs/index.rst [hunk 3]
```diff
@@ -6,7 +6,35 @@
...
+* Population dynamics
+    * The Moran process
+    * An ecological model
+*Multi-processor support, caching for deterministic interactions, automatically
+generate figures and statistics
+
+Every strategy is categorized on a number of dimensions, including:
+    * Deterministic or Stochastic
+    * How many rounds of history used
+    * Whether the strategy has access to the game matrix, the length of the
```
Comment (drvinceknight) [inline docs/index.rst]: whether the strategy makes use of?

The tournament can decide what it has access to right?

**Comment by drvinceknight** [comment]
My PR didn't pick up my inline comments on the outdated diffs. :)

**Comment by marcharper** [comment]
I got them.

File: docs/index.rst [hunk 4]
```diff
@@ -6,7 +6,44 @@
...
+    * An ecological model
+
+* Multi-processor support, caching for deterministic interactions, automatically
+  generate figures and statistics
+
+Every strategy is categorized on a number of dimensions, including:
+
+    * Deterministic or Stochastic
+    * How many rounds of history used
+    * Whether the strategy makes used of the game matrix, the length of the
```
Comment (drvinceknight) [inline docs/index.rst]: makes use

**Comment by drvinceknight** [comment]
👍

**Pull request merged**